packages/core/src/components/banner/banner.global.scss

```diff
@@ -0,0 +1,11 @@
+dso-banner {
+  @include banner_children();
+
+  &[status="danger"] {
+    @include banner_danger();
```

Comment:
Als deze mixin op de juiste plaats staat (nog ff checken): Moeten we deze dan ook prefixen met children?

Reply:
Dat denk ik niet, dit is stijling op de status van de `.dso-banner`, het stijlt niet de kindjes daar binnen (hoewel je misschien het icoon wel als kindje kunt/wilt zien)